ksqldb-serde/src/main/java/io/confluent/ksql/serde/KeyFormat.java

```diff
@@ -24,8 +24,17 @@
 import com.google.errorprone.annotations.Immutable;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import io.confluent.ksql.model.WindowType;
+import io.confluent.ksql.schema.ksql.SchemaConverters;
```

Comment:
These imports look unused? Checkstyle will fail. Looks like there are other unused imports in some other files too -- might want to run checkstyle locally in order to clean these up in order to get a clean Jenkins build sooner :)

Reply:
Sounds good!